config/ember-try.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     {
       name: 'ember-release',
       dependencies: {
-        'ember': 'components/ember#release'
+        'ember': 'components/ember#release',
+        'ember-data': 'v2.0.0-beta.2'
```

Comment:
We can remove ember-data from here as well as `package.json` and `bower.json`, as it is unused by the addon.